### PR TITLE
Add boilerplate update mechanism.

### PR DESCRIPTION
**Pull request by fishinthecalculator**
This is a very basic boilerplate update mechanism. It is designed to simplify the way changes in the build system of Bonfire extensions can be distributed to all extension codebases, but it is definitely lacking in security.

Next steps could be verify a signature but that would require some discussion first probably. Please let me know if you see some simple way to increase security for this.

### CODE REVIEW

**Comment by mayel** [comment]
Looks good to me, not sure what we can do for security, but when we do at least we'll have a mechanism to deploy it to all extensions (since this also updates the update script :) I'd just avoid doing the commit automatically though, as that would also commit other pending changes that the dev maybe wasn't ready to commit, and also to give a chance to review the diff manually before committing (which also helps with security actually).

File: justfile [hunk 1]
```diff
@@ -34,6 +34,12 @@ clean:
     mix deps.clean --all
     rm -rf .hex .mix .cache lib/mix
 
+boilerplate-update:
+    mkdir -p .bonfire-extension-boilerplate
+    git clone https://github.com/bonfire-networks/bonfire-extension-boilerplate.git --branch main --single-branch .bonfire-extension-boilerplate
+    cd .bonfire-extension-boilerplate && cp -r .github lib .envrc justfile ../ && git -C .. add .envrc .github/workflows/main.yml justfile lib/mix/mess.ex && export COMMIT="$(git rev-parse --short HEAD)" && git -C .. commit -m "Update with https://github.com/bonfire-networks/bonfire-extension-boilerplate commit ${COMMIT}"
```
Comment (mayel) [inline justfile]: I shorten the last line to:
`cd .bonfire-extension-boilerplate && cp -r .envrc .github/workflows/main.yml justfile mess.exs`

Comment (mayel) [reply justfile]: (note `mess.exs` rather than `mess.ex` and also it doesn't need to be in `lib/` since it gets compiled manually into `mix.exs`)

Comment (fishinthecalculator) [reply justfile]: Thank you I should have addressed this

**Comment by mayel** [comment]
Looks good! I notice again that you included the more complicate mess from the main app repo, whereas this should only require the simple one used in extensions don't you think?

**Comment by fishinthecalculator** [comment]
> Looks good! I notice again that you included the more complicate mess from the main app repo, whereas this should only require the simple one used in extensions don't you think?

good catch, I forgot. Fixing this as well

**Review by mayel — APPROVED** [review]
♥️

**Pull request merged**